pkgs/development/python-modules/orange3/default.nix

```diff
@@ -0,0 +1,127 @@
+{ lib
+, buildPythonPackage
+, copyDesktopItems
+, fetchurl
+, makeDesktopItem
+, fetchFromGitHub
+, nix-update-script
+, pythonRelaxDepsHook
+, baycomp
+, bottleneck
+, chardet
+, cython
+, httpx
+, joblib
+, keyring
+, keyrings-alt
+, matplotlib
+, numpy
+, openpyxl
+, opentsne
+, orange-canvas-core
+, orange-widget-base
+, pandas
+, pyqtgraph
+, pyqtwebengine
+, python-louvain
+, pyyaml
+, qt5
+, qtconsole
+, requests
+, scikit-learn
+, scipy
+, serverfiles
+, xlrd
+, xlsxwriter
+}:
+
+buildPythonPackage rec {
```

Comment:
This looks like an application, not a library.
If it is not used as a library, we can use　`buildPythonApplication` here and move it to the top level.

Reply:
```
       error: orange3 should use `buildPythonPackage` or `toPythonModule` if it is to be part of the Python packages set.
```

Reply:
I'm not sure what this log is, but this part of the manual may be helpful.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonapplication-function-buildpythonapplication-function